packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -70,17 +71,21 @@
           </div>
         </li>
       </oc-list>
+      <div v-if="imprintUrl && privacyUrl" class="imprint-footer oc-py-s oc-mt-m oc-text-center">
+        <a v-translate :href="imprintUrl" target="_blank">Imprint</a> -
```

Comment:
Please use interpunct (·) as a separator, like in the mockup -> because of beauty. I also highly recommend wrapping that into a span